ZichanYang324/BF-Employee-HR
# BGP-12 Housing Page Frontend - Added housing assignment

# Description
Implemented `assignHousing` enabling housing assignment using the endpoint

Deleted redundant model files.

## Related PRS (if any):

BGP#12

## Main changes explained:
- Added housing assignment


## How to test:

1. check into the current branch
2. Run `npm install` for  backend, then `npm run dev`(Server)
3. use postman to make request to /housing/assign endpoint with body includes `houseId` and `userId` (must have a profile)

## Screenshots or videos of changes or expecting result:


<img width="696" alt="Screen Shot 2024-03-04 at 9 06 40 PM" src="https://github.com/ZichanYang324/BF-Employee-HR/assets/36239681/fba7aaf7-ede4-4c81-a2dd-49e43ecef3e1">


## Note:
Only for demo purpose, no frontend implemented.
